lib/workers/repository/dependency-dashboard.ts

```diff
@@ -16,27 +16,85 @@ import { PackageFiles } from './package-files';
 interface DependencyDashboard {
   dependencyDashboardChecks: Record<string, string>;
   dependencyDashboardRebaseAllOpen: boolean;
+  dependencyDashboardAllPending: boolean;
+  dependencyDashboardAllRateLimited: boolean;
+}
+
+const checkMatchRate = ' - \\[ \\] <!-- unlimit-branch=([^\\s]+) -->';
+const rateLimitedRe = regEx(checkMatchRate, 'g');
+const checkMatchPendingApproval =
+  ' - \\[ \\] <!-- approve-branch=([^\\s]+) -->';
+const pendingApprovalRe = regEx(checkMatchPendingApproval, 'g');
+const checkMatchGeneral = ' <!-- ([a-zA-Z]+)-branch=([^\\s]+) -->';
+const generalBranchRe = regEx(checkMatchGeneral);
+const checkMatchMarkedBranches =
+  ' - \\[x\\] <!-- ([a-zA-Z]+)-branch=([^\\s]+) -->';
+const markedBranchesRe = regEx(checkMatchMarkedBranches, 'g');
```

Comment:
```suggestion
const rateLimitedRe = regEx(' - \\[ \\] <!-- unlimit-branch=([^\\s]+) -->', 'g');
const pendingApprovalRe = regEx(' - \\[ \\] <!-- approve-branch=([^\\s]+) -->', 'g');
const generalBranchRe = regEx(' <!-- ([a-zA-Z]+)-branch=([^\\s]+) -->');
const markedBranchesRe = regEx(' - \\[x\\] <!-- ([a-zA-Z]+)-branch=([^\\s]+) -->', 'g');
```

inline static strings, makes it more readable. we also don't use the strings anywhre else.